src/components/ExperienceDetail/Article/ReactionZone.js

```diff
@@ -70,6 +73,13 @@ const ReactionZone = ({ experienceId, onClickMsgButton }) => {
       >
         留言
       </ReactionButton>
+      <ReactionButton className={styles.report} Icon={() => null}>
```

Comment:
建議直接修改 `ReactionButton` 讓 Icon 為 optional
```suggestion
      <ReactionButton className={styles.report}>
```